app-frontend/src/app/components/leafletMap/leafletMap.controller.js

```diff
@@ -13,12 +13,7 @@ export default class LeafletMapController {
     }
 
     $onInit() {
-        if (this.proposedBounds) {
-            this.map.fitBounds(this.proposedBounds);
```

Comment:
Is it still possible to zoom into a particular bounding box if that bounding box is passed in on page load as a query parameter? I.e. if you load the page, zoom in a bunch, and then open a new tab and copy-paste in the previous URL, do you end up at the same spot?